client/onboarding-prototype/steps/test/loading.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import apiFetch from '@wordpress/api-fetch';
+/**
+ * Internal dependencies
+ */
+import Loading from '../loading';
+
+// Mock Api Fetch module and function
+jest.mock( '@wordpress/api-fetch', () => jest.fn() );
+const mockApiFetch = apiFetch as jest.MockedFunction< typeof apiFetch >;
```

Comment:
We can get rid of that type assertion using `mocked` from `ts-jest/utils`. We just need to wrap it and call any mock method like `mocked( apiFetch ).mockResolvedValueOnce`

Reply:
Thanks for the tip! Addressed in https://github.com/Automattic/woocommerce-payments/pull/5779/commits/266a0ec8a7f67a19dc3676be032035691c39785d

Reply:
Anytime! We only need to wrap it to use mocked methods without TS complaining about it. For expect we don't need to wrap it 🙂

Reply:
Gotcha! Fixed https://github.com/Automattic/woocommerce-payments/pull/5779/commits/de94f6da03dc6287416883ee7806606583e0439d